src/main/java/seedu/address/logic/Warning.java

```diff
@@ -0,0 +1,20 @@
+package seedu.address.logic;
+
+import java.util.Set;
+
+/**
+ * Encapsulates a warning message, containing potentially multiple warnings. Serves a purpose similar to exceptions,
+ * but without terminating the current method.
+ */
+public class Warning {
+    private Set<String> warnings;
```

Comment:
With reference to naming convention on plural vs singular, this might create confusion when referring to warning: are we referring to a single warning string, or a warning object which contains multiple warning strings?

Reply:
Renamed, classes only contain the attribute if needed (as for e.g. warnings will never be needed for list, since warnings are due to user misinput)

Reply:
ah got it, yep sounds good!